src/core/net/nat64_translator.cpp

```diff
@@ -552,8 +700,10 @@ void Translator::ClearNat64Prefix(void)
 
 void Translator::HandleMappingExpirerTimer(void)
 {
-    LogInfo("Released %d expired mappings", ReleaseExpiredMappings());
-    mMappingExpirerTimer.Start(kAddressMappingIdleTimeoutMsec);
+    uint16_t nbOfMappings = ReleaseExpiredMappings();
+    LogInfo("Released %d expired mappings", nbOfMappings);
```

Comment:
This fixes a bug. `LogInfo("Release %d expired mapping", ReleaseExpiredMapping())` is incorrect because `LogInfo()` may become an empty macro if logging at the info level is disabled.

Thanks for finding and fixing this. 👍 

I suggest we add this fix as its own smaller PR. This is important and would allow easier cherry-picking of this bug fix. I can help add this (submit it as smaller PR)?

Reply:
Just submitted this PR. Thanks.
- https://github.com/openthread/openthread/pull/11139.

Reply:
Great, I will remove this from my PR and rebase.